### DISCUSSION TITLE
release 2.23.0 does not show up as available when checking in the admin panel

### DISCUSSION THREAD

**Discussion by m-i-k-e-y** [the-djmaze/snappymail#763] [General]
When checking for updates in the admin panel, release 2.23.0 does not show up as available:

![grafik](https://user-images.githubusercontent.com/29158512/206860406-5d402bdf-aa08-4686-9441-776929c4099d.png)

**Comment by cm-schl** [comment]
Hi, can confirm and opened #770 🙂
